reV/supply_curve/points.py

```diff
@@ -454,6 +462,24 @@ def n_gids(self):
 
         return n_gids
 
+    @property
+    def zone_mask(self):
+        """
+        Get the 2D zone mask, where 1 is included and 0 is excluded.
+
+        Returns
+        -------
+        np.ndarray
+        """
+
+        if self._zone_mask is None:
+            return None
+        else:
+            out_of_extent = self._gids.reshape(self._zone_mask.shape) == -1
```

Comment:
Is this redundant? `zone_mask` is only used to zero out values in the `_inclusion_mask`, which alreasy zeros out out-of-bounds gids. Maybe I am missing something?

Reply:
Yup! I think returning `self._zone_mask` is perfect here